src/components/Doc/ComponentDoc.js

```diff
@@ -79,6 +87,37 @@ export const ComponentDoc = ({
         )}
       </Box>
 
+      {designSystemLink && (
+        <ThemeContext.Extend
+          value={{
+            notification: {
+              container: {
+                background: {
+                  color: 'background-contrast',
+                },
+                round: 'small',
+                pad: 'small',
+              },
+              direction: 'row',
+              message: {
+                color: 'text-strong',
+              },
+            },
+          }}
+        >
+          <Notification
+            message="Using Grommet for HPE?"
+            status="info"
+            actions={[
+              {
+                label: `See ${name} examples and guidance.`,
+                href: designSystemLink,
```

Comment:
can we add `target: _blank` and `rel: noreferrer` so that it'll open in a new window?